homework/index.js

```diff
@@ -1,47 +1,136 @@
+  const tr = createAndAppend('tr', tbody);
+  createAndAppend('td', tr, { class: 'label', text: 'Repository : ' });
+  const td = createAndAppend('td', tr);
+  createAndAppend('a', td, {
+    text: repositories.name,
+    href: repositories.html_url,
+    target: '_blank',
+  });
+  const tr1 = createAndAppend('tr', tbody);
+  createAndAppend('td', tr1, { text: 'Description: ', class: 'label' });
+  createAndAppend('td', tr1, {
+    text: repositories.description,
+  });
+  const tr2 = createAndAppend('tr', tbody);
+  createAndAppend('td', tr2, { text: 'Forks :', class: 'label' });
+  createAndAppend('td', tr2, {
+    text: repositories.forks_count,
+  });
+  const dateString = repositories.updated_at;
+  const dateFormat = new Date(dateString);
+  const tr3 = createAndAppend('tr', tbody);
+  createAndAppend('td', tr3, { text: 'Updated :', class: 'label' });
+  createAndAppend('td', tr3, { text: dateFormat.toUTCString() });
+}
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+function listContributors(contributors, parent) {
+  createAndAppend('h3', parent, { class: ' contributor - header', text: 'contributions' });
+  const ul = createAndAppend('ul', parent, { class: 'contributor-list' });
+  contributors.forEach(contributor => {
+    const itemOfContributor = createAndAppend('li', ul, { class: 'contributor-item' });
+    const divInItem = createAndAppend('div', itemOfContributor, {
+      class: 'contributor-container',
+    });
+    const contributorData = createAndAppend('div', itemOfContributor, { class: 'contributor-data' })
+    createAndAppend('img', divInItem, {
+      src: contributor.avatar_url,
+      class: 'contributor-avatar',
+    });
+    createAndAppend('a', contributorData, {
+      text: contributor.login,
+      href: contributor.html_url,
+      target: '_blank',
     });
+    createAndAppend('div', contributorData, {
+      text: contributor.contributions,
+      class: 'contributor-badge',
+    });
+  });
+}
+
+async function fetchRepositoryContributors(url, parent) {
+  try {
+    let info = await fetchJSON(url);
+    let repo = listContributors(info, parent);
+    return repo;
   }
+  catch (err) {
+    return renderError(err.name);
+  }
+}
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+function renderTableAndContributorsInfo(repositories, parent) {
+  const infoDiv = createAndAppend('div', parent, { class: 'tableInfo' });
+  const flexContainer = createAndAppend('div', parent, { class: 'flexContainer' })
+  const repoInfo = createAndAppend('div', flexContainer, { class: 'left-div whiteframe' });
+  const mainDiv = createAndAppend('div', repoInfo, { id: 'container' });
```

Comment:
You do no seem to have a `whiteframe` CSS style defined in your `style.css`. It looks like you peeked at the HTML generated by https://hyf-github.netlify.com/ where this class is used. I have no issue with a bit of  'creative stealing' like this (as long as the bulk of the works is your own), but you should clean up unused bits.

Reply:
good morning Jim 
thank you for the feedback 
I will definitely check my Vscode  ESlint setting ..
I will also check the unused classes or extra unnecessary things in my code ...
I did look at the website and  structural of the html coz i thought it is require from us to copy this website thats why there is in the homework page link to the site .... but be sure the css style it is completely mine and had diffcult time to style responsive web site coz i start with the big screen and not mobile screen like suppose to be ...
another time thanx for your feedback so i can improve myself everytime and i will for sure try the classes ...